CI/samples.ci/client/petstore/java/test-manual/okhttp-gson/JSONTest.java

```diff
@@ -1,202 +0,0 @@
-    }
-
-    @Test
-    public void testCustomDate() throws Exception {
-        final DateTimeFormatter datetimeFormat = DateTimeFormatter.ISO_OFFSET_DATE_TIME.withZone(ZoneId.of("Etc/GMT+2"));
-        final String dateStr = "2015-11-07T14:11:05-02:00";
-        order.setShipDate(datetimeFormat.parse(dateStr, OffsetDateTime.FROM));
-
-        String str = json.serialize(order);
-        Type type = new TypeToken<Order>() { }.getType();
-        Order o = json.deserialize(str, type);
-        assertEquals(dateStr, datetimeFormat.format(o.getShipDate()));
-    }
-
-    @Test
-    public void testByteArrayTypeAdapterSerialization() {
-        // Arrange
-        final String expectedBytesAsString = "Let's pretend this a jpg or something";
-        final byte[] expectedBytes = expectedBytesAsString.getBytes(StandardCharsets.UTF_8);
-
-        // Act
-        String serializedBytesWithQuotes = json.serialize(expectedBytes);
-
-        // Assert
-        String serializedBytes = serializedBytesWithQuotes.substring(1, serializedBytesWithQuotes.length() - 1);
-        if (json.getGson().htmlSafe()) {
-            serializedBytes = serializedBytes.replaceAll("\\\\u003d", "=");
-        }
-        ByteString actualAsByteString = ByteString.decodeBase64(serializedBytes);
-        byte[] actualBytes = actualAsByteString.toByteArray();
-        assertEquals(expectedBytesAsString, new String(actualBytes, StandardCharsets.UTF_8));
-    }
-
-    @Test
-    public void testByteArrayTypeAdapterDeserialization() {
-        // Arrange
-        final String expectedBytesAsString = "Let's pretend this a jpg or something";
-        final byte[] expectedBytes = expectedBytesAsString.getBytes(StandardCharsets.UTF_8);
-        final ByteString expectedByteString = ByteString.of(expectedBytes);
-        final String serializedBytes = expectedByteString.base64();
-        final String serializedBytesWithQuotes = "\"" + serializedBytes + "\"";
-        Type type = new TypeToken<byte[]>() { }.getType();
-
-        // Act
-        byte[] actualDeserializedBytes = json.deserialize(serializedBytesWithQuotes, type);
-
-        // Assert
-        assertEquals(expectedBytesAsString, new String(actualDeserializedBytes, StandardCharsets.UTF_8));
-    }
-
-    // Obtained 22JAN2018 from stackoverflow answer by PuguaSoft https://stackoverflow.com/questions/11399491/java-timezone-offset
-    // Direct link https://stackoverflow.com/a/16680815/3166133
-    public static String getCurrentTimezoneOffset() {
-
-        TimeZone tz = TimeZone.getDefault();
-        Calendar cal = GregorianCalendar.getInstance(tz, Locale.ROOT);
-        int offsetInMillis = tz.getOffset(cal.getTimeInMillis());
-
-        String offset = String.format(Locale.ROOT,"%02d:%02d", Math.abs(offsetInMillis / 3600000), Math.abs((offsetInMillis / 60000) % 60));
-        offset = (offsetInMillis >= 0 ? "+" : "-") + offset;
-
-        return offset;
-    }
-}
```

Comment:
Note: this still exists at https://github.com/OpenAPITools/openapi-generator/blob/ensures/samples/client/petstore/java/okhttp-gson/src/test/java/org/openapitools/client/JSONTest.java